guardrails/run/async_stream_runner.py

```diff
@@ -303,14 +303,8 @@ async def async_call(
             )
         elif api_fn is None:
             raise ValueError("Either API or output must be provided.")
-        elif msg_history:
```

Comment:
this probably needs to come back because were not actually dropping support this release